locales/puppet.pot

```diff
@@ -0,0 +1,128 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2016 Puppet, Inc.
+# This file is distributed under the same license as the Puppet automation framework package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2016.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: Puppet automation framework 4.8.0-51-gefaf423\n"
+"\n"
+"Report-Msgid-Bugs-To: https://tickets.puppetlabs.com\n"
+"POT-Creation-Date: 2016-11-14 15:35-0800\n"
+"PO-Revision-Date: 2016-11-14 15:35-0800\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
+
+msgid "Apache 2 license; see COPYING"
+msgstr ""
+
+msgid "Display Puppet help."
+msgstr ""
+
+msgid "Display help about Puppet subcommands and their actions."
+msgstr ""
+
+msgid "[<subcommand>] [<action>]"
+msgstr ""
+
+msgid "Short help text for the specified subcommand or action."
+msgstr ""
+
+msgid ""
+"      Get help for an action:\n"
+"\n"
+"      $ puppet help\n"
+msgstr ""
+
+msgid "--version VERSION"
+msgstr ""
+
+msgid "The version of the subcommand for which to show help."
+msgstr ""
+
+#. Check our invocation, because we want varargs and can't do defaults
+#. yet.  REVISIT: when we do option defaults, and positional options, we
+#. should rewrite this to use those. --daniel 2011-04-04
+msgid "Puppet help only takes two (optional) arguments: a subcommand and an action"
+msgstr ""
+
+msgid "Version only makes sense when a Faces subcommand is given"
+msgstr ""
+
+msgid "Legacy subcommands don't take actions"
+msgstr ""
+
+msgid ""
+"Could not load help for the application #{applicationname}.\n"
+"Please check the error logs for more information.\n"
+"\n"
+"Detail: \"#{detail.message}\"\n"
+msgstr ""
+
+msgid ""
+"Could not load help for the face #{facename}.\n"
+"Please check the error logs for more information.\n"
+"\n"
+"Detail: \"#{detail.message}\"\n"
+msgstr ""
+
+msgid "\"Unable to load action #{actionname} from #{face}\""
+msgstr ""
+
+#. Return a list of all applications (both legacy and Face applications), along with a summary
+#. of their functionality.
+#. @return [Array] An Array of Arrays.  The outer array contains one entry per application; each
+#. element in the outer array is a pair whose first element is a String containing the application
+#. name, and whose second element is a String containing the summary for that application.
+#. Now we find the line with our summary, extract it, and return it.  This
+#. depends on the implementation coincidence of how our pages are
+#. formatted.  If we can't match the pattern we expect we return the empty
+#. string to ensure we don't blow up in the summary. --daniel 2011-04-11
+msgid "! Subcommand unavailable due to error. Check error logs."
+msgstr ""
+
+#. @api private
+msgid "Any"
+msgstr ""
```

Comment:
The difference when using `_` and a variable name is that a) Ruby accepts an argument without complaining and b) there is no variable assignment since `_` does not have a value. So a) must have a var or code fails b) faster & less memory used and clearly signals variable is not in use.